<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resources/41t58PIC2gf_10241.JPG);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="LoveButton">
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 10pt &quot;ADMUI3Lg&quot;;
color: rgb(170, 0, 255);</string>
            </property>
            <property name="text">
             <string>我喜欢的</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StopButton">
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(170, 0, 255);font: 10pt &quot;ADMUI3Lg&quot;;</string>
            </property>
            <property name="text">
             <string>定时开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StartButton">
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(170, 0, 255);font: 10pt &quot;ADMUI3Lg&quot;;</string>
            </property>
            <property name="text">
             <string>定时停止</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>424</width>
               <height>560</height>
              </rect>
             </property>
             <widget class="QComboBox" name="comboBox_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>500</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/resources/30.png</normaloff>:/resources/30.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/resources/31.png</normaloff>:/resources/31.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/resources/32.png</normaloff>:/resources/32.png</iconset>
               </property>
              </item>
             </widget>
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>500</y>
                <width>41</width>
                <height>41</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(170, 0, 255);
</string>
              </property>
              <property name="text">
               <string>LRC</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_24">
              <property name="geometry">
               <rect>
                <x>310</x>
                <y>500</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(170, 0, 255);</string>
              </property>
              <property name="text">
               <string>最小化</string>
              </property>
             </widget>
             <widget class="QSlider" name="horizontalSlider">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>420</y>
                <width>391</width>
                <height>22</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>391</width>
                <height>22</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>-30</y>
                <width>421</width>
                <height>411</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/resources/0015e77674b7604f-5a10656e6fe3ec5e-12fb3c14cf5b02d655b56ccf2fb8683c.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>460</x>
                <y>281</y>
                <width>16</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>440</y>
                <width>391</width>
                <height>43</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QToolButton" name="LatPieceButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image:url(:/resources/1.png)</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="PlayButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/20.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="PauseButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/21.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="StopPieceButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/23.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="NextPieceButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/22.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="PlaylistButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/14.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="VoiceButton">
                 <property name="minimumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>47</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/resources/24.png);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>400</y>
                <width>401</width>
                <height>22</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>348</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>41</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>41</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>380</y>
                <width>401</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <zorder>label_2</zorder>
             <zorder>layoutWidget</zorder>
             <zorder>layoutWidget</zorder>
             <zorder>comboBox_2</zorder>
             <zorder>pushButton</zorder>
             <zorder>pushButton_24</zorder>
             <zorder>horizontalSlider</zorder>
             <zorder>label</zorder>
             <zorder>label_5</zorder>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuABOUT">
    <property name="title">
     <string>ABOUT</string>
    </property>
    <addaction name="actionHELP"/>
    <addaction name="actionDESIGNER"/>
   </widget>
   <addaction name="menuABOUT"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHELP">
   <property name="text">
    <string>HELP</string>
   </property>
  </action>
  <action name="actionDESIGNER">
   <property name="text">
    <string>DESIGNER</string>
   </property>
  </action>
  <action name="actionSETTING">
   <property name="text">
    <string>SETTING</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
